.github/workflows/github-token-rotation-prod.yml

```
@@ -0,0 +1,96 @@
+          if ! aws sts get-caller-identity &>/dev/null; then
+            echo "Error: Failed to configure AWS credentials"
+            exit 1
+          fi
+
+      - name: Install PyJWT
+        run: |
+          pip3 install "PyJWT==2.8.0"
+          if ! python3 -c "import jwt; print(jwt.__version__)" &>/dev/null; then
+            echo "Error: Failed to install PyJWT"
+            exit 1
+          fi     
```

yamllint

[error] trailing spaces

(trailing-spaces)